modules/apps/export-import/export-import-api/src/main/java/com/liferay/exportimport/configuration/ExportImportServiceConfiguration.java

```diff
@@ -79,5 +79,15 @@ public interface ExportImportServiceConfiguration {
 		name = "validate-missing-references", required = false
 	)
 	public boolean validateMissingReferences();
+	
+	@Meta.AD(
+			deflt = "15" ,description ="set-lar-cleanup-job-interval-help",
+			name ="set-lar-cleanup-job-interval", required = false )
+	public int setLarCleanupJobInterval();
+	
+	@Meta.AD(deflt = "-1" ,description ="set-lar-expiry-in-days-help",
+			name ="set-lar-expiry-in-days", required = false )
```

Comment:
may name it as `export-import-entry-expiry-days` (if we would rename the attribute to `exportImportEntryExpiryDays`)